src/optvar_sup.erl

```diff
@@ -30,6 +30,7 @@ init([]) ->
                  intensity => 0,
                  period => 1},
     ChildSpecs = [],
+    optvar:init(),
```

Comment:
Q: if this supervisor is the owner of the tables, when the application master kills this supervisor, it'll also drop the tables that the waker process _might_ want to write to.

I was thinking if perhaps it would be safer to make the `init` process an heir of this table, but on second thought it might be best to clear the table in that case.  Just the assert [here](https://github.com/emqx/optvar/blob/8eac297771061be8690b127f572ee3948586f564/src/optvar.erl#L193) might return a `noproc` or `optvar_stopped` rather than `optvar_retry` :thinking:

Reply:
`optvar` app should not be restarted while any process waits for a variable.